source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
```

Comment:
Provide a table of query parameters, their names and explanations. There are examples of this elsewhere in the docs.